src/jats/jats-exporter.ts

```diff
@@ -525,52 +534,56 @@ export class JATSExporter {
       }
     }
 
-    if (titles.title) {
+    if (titleNode) {
       const element = this.document.createElement('article-title')
-      this.setTitleContent(element, titles.title)
-      titleGroup.appendChild(element)
-    }
-
-    if (titles.subtitle) {
-      const element = this.document.createElement('subtitle')
-      this.setTitleContent(element, titles.subtitle)
-      titleGroup.appendChild(element)
-    }
-
-    if (titles.runningTitle) {
-      const element = this.document.createElement('alt-title')
-      element.setAttribute('alt-title-type', 'right-running')
-      this.setTitleContent(element, titles.runningTitle)
+      this.setTitleContent(element, titleNode.textContent)
       titleGroup.appendChild(element)
     }
+    // we don't keep subtitle and running title in PM schema, do we need them??
+    // if (titles.subtitle) {
+    //   const element = this.document.createElement('subtitle')
+    //   this.setTitleContent(element, titles.subtitle)
+    //   titleGroup.appendChild(element)
+    // }
 
-    const supplements = [...this.modelMap.values()].filter(
-      (model) => model.objectType === ObjectTypes.Supplement
-    ) as Supplement[] | undefined
-    if (supplements && supplements.length > 0) {
-      for (const supplement of supplements) {
+    // if (titles.runningTitle) {
+    //   const element = this.document.createElement('alt-title')
+    //   element.setAttribute('alt-title-type', 'right-running')
+    //   this.setTitleContent(element, titles.runningTitle)
+    //   titleGroup.appendChild(element)
+    // }
+    const supplementsNodes = findNodesOfType(
+      this.fragment,
+      schema.nodes.supplement
+    )
+    supplementsNodes.forEach((node) => {
+      if (node.type !== schema.nodes.section_title) {
```

Comment:
Is this check needed? You're looking up the nodes by type already.

Reply:
No, you're right.